platform/accessibility/meeting-notes/2019-08.md

```diff
@@ -0,0 +1,29 @@
+# VA 508 Meeting, 15 August, 2019
+
+This document will outline agenda items created or requested by VA Digital Services and the Product Support team. It can be used as a template for upcoming meetings.
+
+## Points of Discussion
+1. VSP received the first feedback from 508 office on the Disability Calculator beta. Some questions and comments:
+	* **Feedback Item 1:** Disability rating is a required field. VSP to enter a ticket and follow up with team to get the asterisk and SR-only text added.
+	* **Feedback Item 2:** VSP will investigate the form error and issue a ticket for having it read properly.
+	* **Feedback Item 3:** Will issue a ticket to remove the error messages on field blur. Accessibility more than compliance. Mark as lower priority.
+	* **Feedback Item 4:** This is a known issue, logged as a [forms fix](https://github.com/department-of-veterans-affairs/va.gov-team/issues/314) in VA.gov-team
+	* IMPROVE: The data entry type (in 10% increments, and instructions) should be presented on the page instead of in error handling.
+1. VSP discovered a bug in our forms system where checkboxes and radio groups will not read the `<legend>` if screen reader users are navigating with the `TAB` key. Turns out the legned must be the first child of the fieldset, or it won't work correctly and throws a Best Practices warning in axe. Logged as a [sitewide improvement](https://github.com/department-of-veterans-affairs/va.gov-team/issues/767) in VA.gov-team repo.
+1. The in-form "Application has been saved" message we discussed in April is in design discussion: [Application saved message could be missed by assistive devices](https://github.com/department-of-veterans-affairs/va.gov-team/issues/182)
```

Comment:
Used (1) twice, but markdown doesn't care and just keeps numbering.  I'm not sure what MD best practice is.